types/zumly/index.d.ts

```diff
@@ -0,0 +1,63 @@
+// Type definitions for zumly 0.9
+// Project: https://zumly.org
+// Definitions by: edwinork <https://github.com/edwinork>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+    /**
+     * Zumly powers your apps with a zoomable user interface (ZUI) taste.
+     */
+    class zumly {
+        /**
+         * Creates a Zumly instance
+         * @params options
+         * @example
+         *  new Zumly({
+         *  mount: '.mount',
+         *  initialView: 'home',
+         *  views: {
+         *   home,
+         *   contact,
+         *   ...
+         *  }
+         *
+         */
+        constructor(options: Zumly.Options);
+
+        init(): Promise<void>;
+
+        zoomLevel(): number;
+
+        zoomIn(el: Element): Promise<void>;
+
+        zoomOut(): void;
+    }
+
+    namespace Zumly {
+        type WithRender = Record<string, unknown> & { render(): Promise<string> };
+
+        interface Options {
+            // Mount DOM Element
+            mount: string;
+            // First rendered view name
+            initialView: string;
+            // Store of all view objects
+            views: Record<string, WithRender | string>;
+            // Custom transitions
+            transitions?: {
+                // Effects for background views
+                effects: Array<'blur' | 'sepia' | 'saturate'>;
+                // How new injected view is adapted. String. Default 'width'
+                cover: 'height' | 'width';
+                // Transition duration. Default '1s'
+                duration: string;
+                // Transition ease. Default 'ease-in-out'
+                ease: string;
+            };
+            // Activate debug notifications
+            debug?: boolean;
+        }
+    }
+}
+
+export default zumly;
```

Comment:
This is a UMD module, meaning it’s available as a global, through CommonJS, or through AMD. You covered the global case manually but the `export default` is wrong. What you want to do is delete the `declare global`, rename the namespace to lowercase `zumly` like the class, replace the `export default` with `export = zumly`, and add `export as namespace zumly` at the end.

Reply:
I appreciate your help! I think I got confused with trying to make it available as both ES6 module and UMD to match [this example](https://github.com/zumly/zumly#setup).

I pushed the changes requested by you. Please take a look when you have time.